creator-node/src/components/replicaSet/URSMRegistrationComponentService.js

```diff
@@ -33,35 +36,16 @@ const { recoverWallet, signatureHasExpired } = require('../../apiSigning')
  * @param {string} reqSignature signature generated by requesting node, used in public key recovery
  */
 const respondToURSMRequestForSignature = async ({ libs: audiusLibs, nodeConfig }, logger, spID, reqTimestamp, reqSignature) => {
-  if (!spID || !reqTimestamp || !reqSignature) {
-    throw new ErrorBadRequest('Must provide all required query parameters: spID, timestamp, signature')
-  }
-
-  spID = parseInt(spID)
-
-  /**
-   * Fetch node info from L1 ServiceProviderFactory for spID
-   */
-  const spRecordFromSPFactory = await audiusLibs.ethContracts.ServiceProviderFactoryClient.getServiceEndpointInfo(
-    'content-node',
-    spID
-  )
-  let {
-    owner: ownerWalletFromSPFactory,
-    delegateOwnerWallet: delegateOwnerWalletFromSPFactory,
-    endpoint: nodeEndpointFromSPFactory
-  } = spRecordFromSPFactory
-  delegateOwnerWalletFromSPFactory = delegateOwnerWalletFromSPFactory.toLowerCase()
-
-  /**
-   * Reject if node is not registered as valid SP on L1 ServiceProviderFactory
-   */
-  if (
-    LibsUtils.isZeroAddress(ownerWalletFromSPFactory) ||
-    LibsUtils.isZeroAddress(delegateOwnerWalletFromSPFactory) ||
-    !nodeEndpointFromSPFactory
-  ) {
-    throw new ErrorBadRequest(`SpID ${spID} is not registered as valid SP on L1 ServiceProviderFactory`)
+  let ownerWalletFromSPFactory, delegateOwnerWalletFromSPFactory, nodeEndpointFromSPFactory
```

Comment:
this function def doesn't work right? these 3 values are never assigned anywhere

Reply:
updated verifyRequesterIsValidSP to return these 3 values